### ISSUE TITLE
Breakpoints are disabled with MSVC LLDB on 2021.1

### ISSUE THREAD

**Issue by Undin** [intellij-rust/intellij-rust#6915]
## Environment

* **IntelliJ Rust plugin version:** 0.3.142.3705-211
* **Rust toolchain version:** 1.50.0 (cb75ad5db 2021-02-10) x86_64-pc-windows-msvc
* **IDE name and version:** IntelliJ IDEA 2021.1 EAP Ultimate Edition (IU-211.6222.4)/CLion 2021.1 EAP (CL-211.6085.16)
* **Operating system:** Windows 10 10.0
* **Macro expansion engine:** new
* **Name resolution engine:** old

## Problem description

![disabled-breakpoint-2021 1-msvc](https://user-images.githubusercontent.com/2539310/109776182-5fe50700-7c13-11eb-87f0-e753eced526e.PNG)

Note, it happens only with MSVC LLDB. In CLion, everything works as expected with MinGW and stable-gnu toolchain

## Steps to reproduce
```rust
use std::io::stdin;

fn main() {
    println!("Hello!");
    let mut buf = String::new();
    stdin().read_line(&mut buf);
    println!("{}", buf);
}
```

**Comment by ortem** [comment]
I've reproduced this. I will investigate the changes contained in the new MSVC LLDB build shipped with 2021.1.

**Comment by Undin** [comment]
Fixed in latest EAPs (211.6432). Thanks @ortem and @aleksandr-urakov
